src/app/[organization]/songs/[songId]/page.tsx

```
@@ -38,89 +26,88 @@ export default async function SetListPage({
-    redirect(`/`);
+  if (!userId) {
+    redirect("/");
   }
 
   const userData = await api.user.getUser({ userId });
   const userMembership = userData?.memberships[0];
 
-  if (!userMembership) {
-    redirect(`/`);
+  const song = await api.song.get({
+    organizationId: userMembership!.organizationId,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)